auth/gcloud/aio/auth/session.py

```diff
@@ -55,6 +59,16 @@ async def close(self) -> None:
 if not BUILD_GCLOUD_REST:
     import aiohttp
 
+
+    async def _raise_for_status(resp: aiohttp.ClientResponse):
```

Comment:
Please add `-> None` to the type signature.

Reply:
Done